### PR TITLE
fix(card): inner wrapper height inherit(VIV-1151)

### PR DESCRIPTION
**Pull request by rachelbt**

### CODE REVIEW

File: libs/components/src/lib/card/README.md [hunk 1]
```diff
@@ -169,3 +169,27 @@ The number of lines is controlled by css variable `--subtitle-line-clamp`.
 >
 </vwc-card>
 ```
+
+## Use case
+```html preview
+<style>
+  vwc-card {}
+</style>
```
Comment (rinaok) [inline libs/components/src/lib/card/README.md:177]: can be removed

Comment (rachelbt) [reply libs/components/src/lib/card/README.md:177]: right! thanks

Comment (rachelbt) [inline libs/components/src/lib/card/README.md:177]:
```suggestion
```

**Comment by rachelbt** [comment]
> vwc-layout should be added to the ui.test.ts

it is there - the width of the ui-test is 400 something thats why they appear one after another...

**Comment by rinaok** [comment]
> > vwc-layout should be added to the ui.test.ts
> 
> it is there - the width of the ui-test is 400 something thats why they appear one after another...

@rachelbt can you make the width bigger?

**Comment by rachelbt** [comment]
> > > vwc-layout should be added to the ui.test.ts
> > 
> > 
> > it is there - the width of the ui-test is 400 something thats why they appear one after another...
> 
> @rachelbt can you make the width bigger?

changes the style of the card in layout and updated the png's

**Pull request merged**